addons/website_event_track/data/event_track_data.xml

```diff
@@ -5,36 +5,63 @@
             <field name="name">Proposal</field>
             <field name="sequence">1</field>
             <field name="color">1</field>
+            <field name="legend_blocked">Blocked</field>
+            <field name="legend_done">Ready for Next Stage</field>
+            <field name="legend_normal">In Progress</field>
+            <field name="is_visible_in_agenda">False</field>
```

Comment:
False will be evaluated like a string, aka True. You should use eval="True" to evaluate True/False. Moreover as False should be the default value, actually no need to specify it.